kafka-streams-partitioners/weighted-group-partitioner/build.gradle.kts

```diff
@@ -10,6 +10,14 @@ tasks.test {
 }
 
 dependencies {
+  constraints {
+    implementation("com.fasterxml.jackson.core:jackson-databind:2.15.2")
```

Comment:
Just like the one below, will be good to have a `because` indicating why we are applying this constraint.